src/app/components/layout/inputs/LimitedTextArea.js

```diff
@@ -53,13 +54,15 @@ const LimitedTextArea = ({
       value={localText}
       helpContent={(
         <>
-          {helpContent && (<>{helpContent}<br /></>)}
-          <FormattedMessage
-            id="limitedTextAreaWithCounter.counter"
-            defaultMessage="{remaining, plural, one {# character left} other {# characters left}}"
-            description="Label that displays how many characters more can be typed"
-            values={{ remaining: maxChars - (localText.length || 0) }}
-          />
+          {helpContent && (<>{helpContent}</>)}
```

Comment:
Could this line simply become `{helpContent}` now? :smile:

Reply:
ha ha...what's wrong @amoedoamorim ?? you don't like to be really really sure about things??
fixed!